src/app/learnocaml_student_view.ml

```diff
@@ -413,8 +413,9 @@ let display_report exo report =
 let update_answer_tab, clear_answer_tab = ace_display El.Tabs.(editor.tab)
 
 let update_draft, clear_draft_tab =
-  let draft_editor = H.div ~a: [] [] in
-  Manip.appendChild El.Tabs.(draft.tab) draft_editor;
+  let draft_editor = find_component "draft_editor" in
+    (*H.div ~a: [] [] in
+  Manip.appendChild El.Tabs.(draft.tab) draft_editor;*)
```

Comment:
Let's avoid adding commented code in patches.

Reply:
I know, this is already solved in the final patch IINM.